react/ListPrice.tsx

```diff
@@ -3,30 +3,39 @@ import { defineMessages } from 'react-intl'
 import { ProductContext } from 'vtex.product-context'
 import { FormattedCurrency } from 'vtex.format-currency'
 import { useCssHandles } from 'vtex.css-handles'
+import { IOMessageWithMarkers } from 'vtex.native-types'
 
 import { StorefrontFC } from './types'
-import IOMessageWithMarkers from './utils/IOMessageWithMarkers'
 
 interface Props {
-  label: string
+  message: string
   markers: string[]
 }
 
-const CSS_HANDLES = ['listPrice'] as const
+const CSS_HANDLES = ['listPrice', 'listPriceValue'] as const
 
 const ListPrice: StorefrontFC<Props> = props => {
-  const { label, markers } = props
-  const { selectedItem } = useContext(ProductContext)
+  const { message, markers } = props
```

Comment:
```suggestion
  const { listPriceLabel, markers } = props
```
I prefer to keep those props with something more specific, because when you go to implement product-best-price there will be two different labels, sellingPriceLabel and bestPriceLabel

Reply:
We agreed to continue with `message` and then change it if necessary before 1.x